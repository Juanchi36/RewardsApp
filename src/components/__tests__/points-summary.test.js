import React from 'react';
import { render } from '@testing-library/react-native';
import { PointsSummary } from '@components';

describe('pointsSummary', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should return a snapshot of the component', () => {
    const result = render(<PointsSummary ammount={10000} month="Diciembre" />);

    const container = result.getByTestId('points-summaty-container');

    expect(container).toBeDefined();
    expect(result.toJSON()).toMatchSnapshot();
  });
});
