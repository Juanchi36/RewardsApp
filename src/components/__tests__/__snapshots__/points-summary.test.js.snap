// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pointsSummary should return a snapshot of the component 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#334ffa",
        "borderRadius": 20,
        "elevation": 1,
        "height": 143,
        "justifyContent": "center",
        "shadowOffset": {
          "height": 4,
          "width": 0,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 20,
      },
    ]
  }
  testID="points-summaty-container"
>
  <Text
    style={
      [
        {
          "color": "#ffffff",
          "fontFamily": "Avenir-Medium",
          "fontSize": 16,
          "fontWeight": "800",
          "left": 18,
          "lineHeight": 21.86,
          "position": "absolute",
          "top": 21,
        },
      ]
    }
  >
    Diciembre
  </Text>
  <Text
    style={
      [
        {
          "color": "#ffffff",
          "fontFamily": "Avenir-Medium",
          "fontSize": 32,
          "fontWeight": "800",
          "lineHeight": 43.71,
          "textAlign": "center",
        },
      ]
    }
  >
    10,000.00 pts
  </Text>
</View>
`;
